cmd/Makefile.docker

```diff
@@ -1,3 +1,5 @@
+CIRCLE_BUILD_NUM ?= 1
+
 containerbin:
 	export PKG_CONFIG_PATH=$(GOPATH)/src/github.com/xtraclabs/cqrs-sample-pub/pkgconfig/
-	go build -o esatomreplicator
+	go build -o -ldflags "-s -X github.com/xtracdev/es-atom-replicator/health.Version=$(CIRCLE_BUILD_NUM)" esatomreplicator
```

Comment:
Might be missing something here but where in the health package is the Version struct that is being set?

Reply:
Just found it - state.go - ignore my last comment :)